src/reformat.c

```diff
@@ -457,6 +457,12 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image,
         uint8_t * ptrG = &rgb->pixels[state->rgbOffsetBytesG + (j * rgb->rowBytes)];
         uint8_t * ptrB = &rgb->pixels[state->rgbOffsetBytesB + (j * rgb->rowBytes)];
 
+        // If toAlphaMode is active (not no-op), assert that the appropriate alpha plane pointer is
+        // present (for this image's depth). The end of the avifPrepareReformatState() function should
+        // ensure this, but this assert makes it clear to clang's analyzer.
+        assert((state->toRGBAlphaMode == AVIF_ALPHA_MULTIPLY_MODE_NO_OP) ||
+               (((image->depth == 8) && ptrA8) || ((image->depth > 8) && ptrA16)));
```

Comment:
Optional: The property ensured by the end of the avifPrepareReformatState() function is:

`assert((state->toRGBAlphaMode == AVIF_ALPHA_MULTIPLY_MODE_NO_OP) || aPlane);`

Would this simpler assertion work? This assertion can be moved outside the for loop, to line 436.